scripts/bioconductor/bioconductor_skeleton.py

```diff
@@ -54,12 +54,10 @@ def setup_logger(loglevel):
 #
 #   conda create -n rtest -c r r
 #   R -e "rownames(installed.packages())"
-BASE_R_PACKAGES = ["base", "boot", "class", "cluster", "codetools", "compiler",
-                   "datasets", "foreign", "graphics", "grDevices", "grid",
-                   "KernSmooth", "lattice", "MASS", "Matrix", "methods",
-                   "mgcv", "nlme", "nnet", "parallel", "rpart", "spatial",
-                   "splines", "stats", "stats4", "survival", "tcltk", "tools",
-                   "utils"]
+BASE_R_PACKAGES = ["base", "compiler", "datasets", "graphics", "grDevices",
```

Comment:
This is really great - they changed the build-in packages in a point release!
I have seen this as well and fixed a few packages already. Thanks for formalizing it.